jsx-runtime/package.json

```diff
@@ -3,6 +3,7 @@
   "amdName": "jsxRuntime",
   "version": "1.0.0",
   "private": true,
+  "sideEffects": false,
```

Comment:
Just noticed we were missing this, this might make the `jsx` stuff tree shake out and same for the `IS_NON_DIMENSIONAL`